hive_agent/server/routes/chat.py

```diff
@@ -45,64 +48,18 @@ async def event_generator():
                     break
                 yield token
 
-        if isinstance(chat_manager.llm, OpenAIAgent):
-            response = await chat_manager.llm.astream_chat(last_message_content, messages)
-            return StreamingResponse(event_generator(), media_type="text/plain")
-        return None
-
-    async def generate_response_or_stream(
-        chat_manager: ChatManager,
-        db_manager: DatabaseManager,
-        request: Request,
-        last_chat_message: ChatMessage,
-        messages: List[ChatMessage],
-        file_paths: Optional[List[str]] = None,
-    ):
-        llm_response = await process_llm_response(request, chat_manager, str(last_chat_message.content), messages)
-        if llm_response:
-            return llm_response
-        return await chat_manager.generate_response(db_manager, messages, last_chat_message, file_paths or [])
+        # if isinstance(chat_manager.llm, OpenAIAgent):
+        response = await chat_manager.llm.astream_chat(last_message_content, messages)
```

Comment:
this won't get save to the db. it's better not to call the llm directly but to use the method on `ChatManager`